rrfs-test/validated_yamls/templates/obtype_config/ztd_ukmo.yaml

```diff
@@ -0,0 +1,70 @@
+     - obs space:
+         name: Groundgnss  
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: single_ztd_obs_2024052700.nc
+#            obsfile: ioda/2024052700.rap.t00z.ztd.tm00.bufr_d.nc
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: ztd_single_diag_out_2024052700.nc4
```

Comment:
This one would need to also be updated with the current norms. This would be something like:
```yaml
             obsfile: jdiag_gnss_zenithTotalDelay_123.nc4
```
or whatever you decide on the `obs space` name.

Reply:
updated.